Tests/NIOHTTPCompressionTests/HTTPResponseCompressorTest.swift

```diff
@@ -525,10 +525,8 @@ class HTTPResponseCompressorTest: XCTestCase {
         }
 
         XCTAssertNoThrow(try channel.pipeline.removeHandler(name: "compressor").wait())
-
-        do {
-            try writePromise.futureResult.wait()
-        } catch {
+        
+        XCTAssertThrowsError(try writePromise.futureResult.wait()) {error in
```

Comment:
This change is not quite right: we don't _know_ that this will throw an error, nor do we assert it must, it just _might_ and we want to swallow that. Please revert.

Reply:
Okay I have reverted this change.